addons/product/product_view.xml

```diff
@@ -108,7 +108,7 @@
                                     </group>
                                     <group name="group_standard_price">
                                         <field name="list_price"/>
```

Comment:
Oh my god, I must have been sleeping while doing that. So sorry, I'll correct it